features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Create/NewApp.jsx

```diff
@@ -82,12 +82,16 @@ class NewApp extends React.Component {
                 throttlingPolicy: '',
                 description: '',
                 tokenType: 'OAUTH',
+                groups: null,
                 attributes: {},
             },
             isNameValid: true,
             throttlingPolicyList: [],
             allAppAttributes: null,
+            isApplicationSharingEnabled: true,
         };
+        this.handleAddChip = this.handleAddChip.bind(props);
```

Comment:
Shall we bind to `this` rather than binding to `props`?

Reply:
changed to 'this' and resolved.